vestido/vestido-storefront/modules/Checkout/CheckoutView.tsx

```diff
@@ -224,25 +241,58 @@ const CheckoutView: React.FC = () => {
-              <div className="flex flex-col">
-                {checkoutItems?.map((checkoutItem, index) => (
-                  <div key={index}>
-                    <div className="flex justify-between py-3 px-2 gap-2 items-center">
-                      <ItemImage
-                        item={checkoutItem.item}
-                        width={50}
-                        height={70}
-                        className="w-10 h-12 col-span-1"
-                      />
-                      <div className="text-sm col-span-3 text-left grow pl-5">
-                        {checkoutItem.item.title}
-                      </div>
-                      <div className="text-sm col-span-1 flex justify-center text-right">
-                        ₹&nbsp;{checkoutItem.item.price.toFixed(2)}
-                      </div>
-                    </div>
-                  </div>
-                ))}
+              <Table>
+                <TableHeader>
+                  <TableRow className="border-none hover:bg-transparent">
+                    <TableHead>Image</TableHead>
+                    <TableHead>Item</TableHead>
+                    <TableHead>Qty</TableHead>
+                    <TableHead className="text-right">Amount</TableHead>
+                  </TableRow>
+                </TableHeader>
+
+                <TableBody>
+                  {checkoutItems?.map((checkoutItem, index) => {
+                    const itemAmount = (
+                      checkoutItem.item.price * checkoutItem.qty
+                    ).toFixed(2);
+                    return (
+                      <TableRow
+                        key={index}
+                        className="border-none hover:bg-transparent"
+                      >
+                        <TableCell>
+                          <ItemImage
+                            item={checkoutItem.item}
+                            width={50}
+                            height={70}
+                            className="w-10 h-12 col-span-1"
+                          />
+                        </TableCell>
+                        <TableCell>{checkoutItem.item.title}</TableCell>
+                        <TableCell>{checkoutItem.qty}</TableCell>
+                        <TableCell className="text-right">
+                          ₹&nbsp;{itemAmount}
+                        </TableCell>
+                      </TableRow>
+                    );
+                  })}
+                </TableBody>
+              </Table>
+              <hr className="border-gray-600" />
+              <div className="flex gap-10 my-3">
+                <InputElement
+                  name="couponCode"
+                  inputRef={couponInputRef}
+                  placeholder="Enter Coupon Code"
+                  className="bg-gray-700 focus-visible:ring-0 border-none focus-visible:ring-offset-0 ring-offset-black"
+                />
```

Comment:
You cannot have InputElement here. It will automatically update `react-hook-forms`'s state. We want to delay updating it till the user clicks on the button. Better to use normal Button.